box-share-sample/src/main/java/com/box/androidsdk/sample/MainActivity.java

```diff
@@ -66,6 +76,18 @@ protected void onSaveInstanceState(Bundle outState) {
     }
 
     private void initialize(){
+        Typeface font = Typeface.createFromAsset(getAssets(), "Roboto-Medium.ttf");
```

Comment:
I would suggest not customizing fonts/doing other styling in code, since this adds to all the code the developers have to read - when they are trying to see how to use our sdk.